spikeinterface/widgets/ipywidgets/amplitudes.py

```diff
@@ -34,6 +34,7 @@ def do_plot(self, data_plot, **backend_kwargs):
                 fig = plt.figure(figsize=((ratios[1] * width_cm) * cm, height_cm * cm))
                 plt.show()
 
+        data_plot['unit_ids'] = [data_plot['unit_ids'][0]]
```

Comment:
This is very very weird weird and hard to understand.
This sounds like a design problem. No ?

Reply:
No, the problem is that by default when `unit_ids` is None, all of them are selected. With the `ipywidgets` backend is better to initialize with one unit, and the user can select anything later. 

If you have hundreds of units it doesn't make sense to plot everything at once (but that's default)